src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/InsertOneTableIntegrationTest.java

```diff
@@ -48,6 +49,18 @@ public final void createDefaultTables() {
             "bigIntegerValue",
             Map.of("type", "varint")),
         "id");
+    createTableWithColumns(
```

Comment:
So for Integration test, we still need to create the table with correct columns to test against, correct?
The previous thing Aaron showed for all kinds of code combination, that is unit test?

Reply:
Correct on both: Aaron has been writing unit tests, with more scaffolding. I added some unit tests too, but those are simpler parameterized codec tests.

ITs do use some new helper classes but do require old-style table creation.
We will probably need a way to create tables with direct CQL but for now use of `createTable` works.